example/credentials.env

```diff
@@ -0,0 +1,2 @@
+siteId=SITE_ID
```

Comment:
I think this file should:
1. be added to `.gitignore`
2. have a `credentials.env.example` file created that shows the expected format of the file. 

It would be pretty easy for someone to accidentally add this file to version control by accident with real creds.